.github/workflows/test-e2e-pr.yml

```
@@ -4,106 +4,175 @@ on:
 
-      - name: Upload PR number
-        if: ${{ steps.parseTitle.outputs.tip }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: pr
-          path: ./pr-id.txt
+      # 解析测试命令配置
       - name: Parse Test Cli
-        id: parsetestCli
+        id: parse-test-cli
         uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)